__tests__/app.test.js

```diff
@@ -125,4 +134,78 @@ describe("app.js", () => {
       expect(message).toBe("Invalid datatype for parameter");
     });
   });
+  describe("/api/articles/:article_id/comments", () => {
+    test("POST 201: responds to the client with an object containing the username and body of the comment", async () => {
+      const commentToPost = { username: "lurker", body: "I agree!" };
+      const response = await request(app)
+        .post("/api/articles/13/comments")
+        .send(commentToPost)
+        .expect(201);
+      const { body } = response;
+      const { comment } = body;
+      expect(Object.keys(comment).length).toBe(6);
+      expect(comment).toMatchObject({
+        comment_id: 19,
+        body: "I agree!",
+        article_id: 13,
+        author: "lurker",
+        votes: 0,
+      });
+      expect(typeof comment.created_at).toBe("string");
+      const result = await db.query(
+        `SELECT * FROM comments WHERE article_id = $1`,
+        [13]
```

Comment:
This is an interesting way to gain extra confidence that it was posted, I wouldn't say it's necessary to be this secure but I really like the approach nonetheless! You could skip using the `$1` and just put `13` directly in, as this query is only being made by the test so it isn't open to SQL injection by a user